### PR TITLE
add lerna-changelog and release it

### PR DESCRIPTION
**Pull request by Mikek2252**
This PR adds release it and changelog to manage creating releases.

As an older version of node is require i have also pinned a node version with volta

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,18 +20,24 @@
...
   "devDependencies": {
+    "@release-it-plugins/lerna-changelog": "^5.0.0",
     "coveralls": "^3.1.0",
     "eslint": "^8.6.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.0.0",
     "jest": "^26.6.3",
-    "prettier": "^2.5.1"
+    "lerna-changelog": "^2.2.0",
```
Comment (oscard0m) [inline package.json]: Is `lerna-changelog` dependency necessary? I see `@release-it-plugins/lerna-changelog` [has it as a dependency already](https://github.com/release-it-plugins/lerna-changelog/blob/8a5d744139d1613af170f24fdad90dea6523b347/package.json#L35).

```suggestion
```

File: release-it.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    tagName: 'v${version}',
+  },
+  github: {
+    release: true,
+    releaseName: 'v${version}',
+    tokenRef: 'GITHUB_AUTH',
+  },
+  npm: {
+    publish: false,
+  },
```
Comment (oscard0m) [inline release-it.js:18]: For my learning and understanding, why do we plan to publish in GitHub's Registry and not in NPM?

**Review by oscard0m — APPROVED** [review]
LGTM!

File: release-it.js (hunk 2 above)
Comment (oscard0m) [reply release-it.js:18]: Ok now I see this PR: https://github.com/simplabs/ember2-x-codemods/pull/5/files, so the workflow is to publish on GitHub manually (and test the package?) before publishing it into NPM?

Comment (Mikek2252) [reply release-it.js:18]: Yeah so it runs the test workflow before publishing

**Pull request merged**